setup.py

```diff
@@ -5,7 +5,7 @@
 import sys
 
 extra_install_requires = []
-extra_tests_require = []
+extra_tests_require = ['httpretty']
```

Comment:
Ah cool, updated. I've also changed `extra_*_requires` to `pyver_*_requires`. I don't think docs are actually necessary as the name change makes it pretty obvious.